File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L3/08/Kasus2L3.py
import math

# Enter radius of the cylinder
print("Enter the radius and length of a cylinder: ", end="")
radius, length = map(float, input().split())

# Calculate Area
area = radius * radius * math.pi

# Print area cylinder
print("The area is", area)

# Calculate Volume
volume = area * length

# Print volume cylinder
print("The volume of the cylinder is", volume)

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/plagiarized/L3/08/Kasus2L3.kt
package IR

import java.util.Scanner

fun main(args: Array<String>) {
    // Enter radius of the cylinder
    println("Enter the radius and length of a cylinder: ")
    val scanner = Scanner(System.`in`)
    val radius = scanner.nextDouble()
    val length = scanner.nextDouble()
    // Hitung Area
    val luas = radius * radius * 3.14159
    // Print area cylinder
    println("The area is $luas")
    //Hitung Volume
    val volume = luas * length
    // Print volume cylinder
    println("The volume of the cylinder is $volume")
}

